packages/frontend/core/src/components/affine/page-properties/info-modal/info-modal.tsx

```diff
@@ -63,14 +75,19 @@ export const InfoModal = ({
         'aria-describedby': undefined,
       }}
       open={open}
-      onOpenChange={onOpenChange}
+      onOpenChange={handleOpenChange}
       withoutCloseButton
     >
       <Scrollable.Root>
         <Scrollable.Viewport
           className={styles.viewport}
           data-testid="info-modal"
         >
+          <input
```

Comment:
sorry I don't understand this fix. Can you please explain?

Reply:
Before the modification, when the modal is closed, if a value is being edited, the input's on blur event cannot be triggered to automatically save.
The way I handle this issue is to create a hidden input and automatically focus it before closing the modal, thus triggering a lost focus event for the other input.

Reply:
I see. Not sure if this fix is too hacky. Maybe adding an exiting animation could also work?

Reply:
If animation is necessary, I think it's better to implement in global `<Modal />` and make it configurable.